### DISCUSSION TITLE
Customize Google Hangout Chat alert message

### DISCUSSION THREAD

**Discussion by thequailman** [grafana/grafana#38122] [Alerting]
**What would you like to be added**:
Please let us customize the entire text of the Google Hangout Chat alert message using the alert template format.

**Why is this needed**:
There is no way to customize the alert message format for this contact point.

**Comment by alexweav** [comment]
Hi :wave: I posted a PR for this: #43568

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
